spec/reducer_spec.rb

```diff
@@ -64,10 +64,17 @@
     expect(Fixtures::ArtistReducer.apply(nil)).to be_truthy
   end
 
+  it 'applies default filters' do
+    get '/artists' do |response|
+      name = response.json[0]['name']
+      expect(name).to eq('Björk')
+    end
+  end
+
   it 'can sort' do
-    get '/artists?order=genre' do |response|
+    get '/artists?sort=genre' do |response|
```

Comment:
FYI: That test request param was wrong. `order` instead of `sort`, see fixture. Fixed it.